docs/app/Examples/elements/Header/Types/HeaderSubHeadersExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderSubHeadersExamples extends Component {
+  render() {
+    return (
+      <Header.H2 className='sub'>
```

Comment:
Oh, I see.  Just realized for the first time that there is a `ui sub header` type that takes a `sub header` child for this case.  Thanks.